package.json

```diff
@@ -110,6 +110,7 @@
     "google-auth-library": "^6.1.3",
     "inquirer": "~6.3.1",
     "js-yaml": "^3.13.1",
+    "JSONStream": "^1.2.1",
```

Comment:
What's with this?

Reply:
Annoying sorting artifact based on npm version